third_party/intel/lib/TritonGENToLLVM/TritonGENToLLVMPass.cpp

```diff
@@ -245,12 +245,60 @@ static bool isOCLBuiltinAvailable(TritonGEN::Matrix2DBlockLoadOp op) {
   if (op.getVBlocks() != 2)
     return false;
 
-  if (op.getCacheControl() != TritonGEN::LoadCacheControl::DEFAULT)
-    return false;
-
   return true;
 }
 
+static SmallVector<Attribute>
+loadCacheControlToDecoration(Builder &builder, uint32_t operandNum,
+                             TritonGEN::LoadCacheControl orig) {
+  const auto build = [&builder,
+                      operandNum](TritonGEN::LoadCacheControlDecorationEnum l1,
+                                  TritonGEN::LoadCacheControlDecorationEnum l3)
+      -> SmallVector<Attribute> {
+    return {builder.getAttr<TritonGEN::LoadCacheControlDecorationAttr>(
+                1, l1, operandNum),
+            builder.getAttr<TritonGEN::LoadCacheControlDecorationAttr>(
+                3, l3, operandNum)};
```

Comment:
on PVC, SPIRV cache level 0 maps to L1 and cache level 1 maps to L3.
```suggestion
                0, l1, operandNum),
            builder.getAttr<TritonGEN::LoadCacheControlDecorationAttr>(
                1, l3, operandNum)};
```